src/test/scala/io/dockstore/release_1_9/Organizations.scala

```diff
@@ -0,0 +1,54 @@
+package io.dockstore.release_1_9
+
+import io.dockstore.Requests._
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+import scala.util.Random
+
+/**
+  * Simulates a Dockstore user visiting organization and collection pages. Assumes the db that has the orgs and collections listed below.
+  * Chose the broad because it's the most popular and causes us the most problems...
+  */
+object Organizations {
+
+  // Hope to make more random, but for now pick the worst case scenario: Looking at the broad org and gatk collection
+  val organizationList = Array("BroadInstitute", "PCAWG")
+  val broadInstituteCollections = Array("GATKWorkflows", "pgs")
+  val pcawgCollections = Array("PCAWG")
+  var orgName = "BroadInstitute"
```

Comment:
A Scala best practice would be to make this and the ones below `val` instead of `var`. The difference is that `val` is immutable, and Scala strongly encourages immutability.

It's a small thing in this context, but if I see `val`, I know the value isn't changing anywhere below, whereas when I see `var`, I scan the code to see where/how it is modified.